### PR TITLE
Important events

### PR DESCRIPTION
**Pull request by langsharpe**

### CODE REVIEW

File: memoir.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # early childhood
 #
+# rails girls presentation
+#
 # the twilight years
+
```
Comment (jamesottaway) [inline memoir.txt:6]: Pfft, trailing newline.

Comment (langsharpe) [inline memoir.txt:6]: Please keep up to date with our style guide.

Comment (jamesottaway) [inline memoir.txt:6]: :fire: it down!

**Comment by langsharpe** [comment]
Fine. _Storms off_

**Pull request closed without merging**